test/onnx/test_fx_to_onnx.py

```diff
@@ -142,7 +143,25 @@ def forward(self, input, emb):
             expected_error_node="aten.embedding.default",
         )
 
-    def test_unsupported_function_schema_with_op_level_debug(self):
+    @parameterized.expand(
+        [
+            (
+                "bypassing_failed_op_level_debug_raises_diagnostic_warning",
```

Comment:
```suggestion
                "bypassing_failed_op_level_debug",
```

sorry, actually this should be good since "raise_diagnostic_warning" is in the `test_unsupported_function_schema_raise_diagnostic_warning` name already